backend/ee/danswer/background/celery_utils.py

```diff
@@ -12,7 +12,12 @@
 from danswer.db.tasks import get_latest_task
```

Comment:
Everything in this module is just refactoring the group and doc permission syncing to be seperate